server/src/middlewares/user_auth.middleware.js

```diff
@@ -0,0 +1,102 @@
+const axios = require("axios");
+const CustomError = require("../utils/custom-error");
+const url = "https://api.zuri.chat/users/";
```

Comment:
Use the url from the config.js file. Don’t hard code it, it might change in the future